src/test/java/seedu/address/logic/commands/SortStrategyTest.java

```diff
@@ -0,0 +1,110 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.commands.sortfunctions.SortByName;
+import seedu.address.logic.commands.sortfunctions.SortByTag;
+import seedu.address.logic.commands.sortfunctions.SortCommand;
+import seedu.address.logic.commands.sortfunctions.SortStrategy;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.AddressBook;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+
+public class SortStrategyTest {
+    @Test
+    public void testSort_success() {
```

Comment:
Just a nit pick, maybe can separate sortTag and sortName into two functions. Otherwise, LGTM.